queries/c_sharp/indents.scm

```diff
@@ -0,0 +1,117 @@
+[
+ (property_declaration)
+ ; (field_declaration)
+ (variable_declaration)
+ (accessor_declaration)
+
+ ; (declaration_list)
+
+ (parameter_list)
+ (type_parameter_list)
+ (accessor_list)
+ (attribute_argument_list)
+ (base_list)
+ (argument_list)
+
+ (binary_expression)
+ (expression_statement)
+ (return_statement)
+ (local_declaration_statement)
+ (local_function_statement)
+ (yield_statement)
+ (fixed_statement)
+ (unsafe_statement)
+ (labeled_statement)
+ (for_each_statement)
+ (for_statement)
+ (if_statement)
+ (switch_expression)
+ (switch_section)
+ (switch_statement)
+ (while_statement)
+ (lambda_expression)
+ (do_statement)
+ (checked_statement)
+ (try_statement)
+ (using_statement)
+ (local_function_statement)
+ (object_creation_expression)
+ (array_creation_expression)
+ (implicit_object_creation_expression)
+
+ ] @indent.begin
+
+(method_declaration body: _ @indent.begin)
+(namespace_declaration body: _ @indent.begin)
+(record_declaration body: _ @indent.begin)
+(class_declaration body: _ @indent.begin)
+(struct_declaration body: _ @indent.begin)
+(constructor_declaration body: _ @indent.begin)
+(interface_declaration body: _ @indent.begin)
+(enum_declaration body: _ @indent.begin)
```

Comment:
Why these and not `(block) @indent.begin`

Reply:
Okay I can see better why this is declared as such. I'd still prefer to capture the node associated to `body` field instead

Reply:
yeah, classes, structs and similar have `declaration_list` in their bodies, but e.g. methods and constructors can have `block` or `arrow_expression_clause`